h2o-algos/src/test/java/hex/util/AggregatorTest.java

```diff
@@ -468,4 +468,25 @@ public void checkNumExemplars(AggregatorModel m) {
     checkNumExemplars(agg);
     agg.remove();
   }
+
+  @Test public void testCovtypeMapping() {
+    Frame frame = parse_test_file("smalldata/covtype/covtype.20k.data");
+
+    AggregatorModel.AggregatorParameters parms = new AggregatorModel.AggregatorParameters();
```

Comment:
I think it would be a good idea to use try-finally block here to avoid leaking keys.